### PR TITLE
Assorted improvements

### PR DESCRIPTION
**Pull request by weijia-cheng**

### CODE REVIEW

File: src/epub/css/core.css [hunk 1]
```diff
@@ -99,12 +99,6 @@ i > em{
 	font-style: normal;
 }
 
-ol,
```
Comment (acabal) [inline src/epub/css/core.css:102]: This block was added as part of the new core.css, can you restore it?

**Review by acabal — COMMENTED** [review]
Great work! I have a few comments, see above

File: src/epub/css/local.css [hunk 2]
```diff
@@ -41,10 +46,6 @@ section[epub|type~="dedication"] b{
 	text-indent: 0;
 }
 
-[epub|type~="z3998:poem"] p + p{
```
Comment (acabal) [inline src/epub/css/local.css:44]: If this selector is unused, why didn't lint catch it? Are you sure it's unused?

Comment (weijia-cheng) [reply src/epub/css/local.css:44]: Actually, lint did catch it. This CSS became unused after I tweaked the poems in b19df3f.

File: src/epub/css/core.css (hunk 1 above)
Comment (weijia-cheng) [reply src/epub/css/core.css:102]: Oops, didn’t realize that. I’ll revert.

**Comment by acabal** [comment]
Thanks!

**Pull request merged**